packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -56,23 +70,36 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
             this._paypalCommerceSdk = await this._paypalScriptLoader.getPayPalSDK(paymentMethod, currencyCode, paypalcommerce.initializesOnCheckoutPage);
         }
 
-        this._renderButton(containerId, methodId, paypalcommerce);
+        this._renderButton(containerId, methodId, paypalcommerce, isHostedCheckoutEnabled);
     }
 
     deinitialize(): Promise<void> {
         return Promise.resolve();
     }
 
-    private _renderButton(containerId: string, methodId: string, paypalcommerce: PaypalCommerceButtonInitializeOptions): void {
+    private _renderButton(containerId: string, methodId: string, paypalcommerce: PaypalCommerceButtonInitializeOptions, isHostedCheckoutEnabled?: boolean): void {
         const { buyNowInitializeOptions, initializesOnCheckoutPage, style } = paypalcommerce;
         const paypalCommerceSdk = this._getPayPalCommerceSdkOrThrow();
+        const hostedCallbacks = {
```

Comment:
It might be better to rename the variable from `hostedCallbacks` to `hostedCheckoutCallbacks`, because shows to the developer that this callbacks needed for `HostedCheckout` flow

Reply:
renamed